### ISSUE TITLE
Not able to type into a Mautic Form Input

### ISSUE THREAD

**Issue by escopecz** [mautic/mautic#164]
If I open this testing page:

http://m.johnlinhart.com/mauve

in a Chrome, I can't type into the form field. It works in FF. I'm on Ubuntu. Is that issue for every Chrome user? Can't find the reason why it is happening...

**Comment by drmmr763** [comment]
Looks to be an issue with the form CSS. Had the same issue on Chrome / OSX. In the inspector window I changed this:

```
.mauticform-input {
  height: 75px !important; // this was 30px
}
```

**Comment by alanhartless** [comment]
Fixed with 1.0
